examples/other_datasets/README.txt

```diff
@@ -0,0 +1,4 @@
+Classification on other datasets
+--------------------------------
+
+Contains examples with other datasets, including examples with non-EEG datasets (e.g. the titanic dataset).
```

Comment:
In fact, `multiclass_classification.py` is an EEG-based example.
Do we split EEG examples according to the BCI paradigm (ERP, MI, etc)?